README.md

```diff
@@ -44,6 +44,27 @@ This include packages related to both motion and process planning for the Tesser
 * [Doxygen](https://ros-industrial-consortium.github.io/tesseract_planning/)
 * [Benchmark](https://ros-industrial-consortium.github.io/tesseract_planning/dev/bench)
 
+## Process Planning Server Tuning
+The planning server is built leveraging the library taskflow which enable you to quickly write parallel and heterogeneous tasks programs which aligns well with process motion planning. These types of libraries leverage thread pools to quickly spinning up different tasks as needed. One interesting thing found, if a task consumes a lot of memory libc will make performance decision to allow the thread to hold onto the memory it has for subsequent tasks opposed to giving the memory back to your system. This may be fine if this was the only set of tasks being ran but typically in our use case there are other computationally intensive processes which could use this memory.
```

Comment:
You might give this a proofread. Here are a few things.

`which enable` -> `which enables`
`to quickly spinning` -> `to quickly spin`
`libc will make performance` -> `libc will make a performance`
`tasks being ran` -> `tasks being run'